docs-2.0/3.ngql-guide/6.functions-and-expressions/1.math.md

```diff
@@ -46,3 +48,23 @@ Nebula Graph支持以下内置数学函数。
 !!! Note
 
     如果参数为`NULL`，则输出结果是未定义的。
+
+## 示例
+
+```ngql
+# 支持聚合函数
+nebula>  GO FROM "Tim Duncan" OVER like YIELD like._dst AS dst, $$.player.age AS age \
+         | GROUP BY $-.dst \
+         YIELD \
+         $-.dst AS dst, \
+         (INT)(sum($-.age)/count($-.age))+avg(distinct $-.age+1)+1 AS statistics
```

Comment:
DO NOT USE C style cast

use toInteger() instead.

Reply:
OK, I have fixed it.

Reply:
This function has no specific meaning, which means that you can use an aggregate function like "COUNT(v)+AVG(v)"